liteflow-labs/starter-kit
# Collection Detail Page

### Project organization

- Trello card: https://trello.com/c/EDKgv1pH
- Dependant on https://github.com/liteflow-labs/starter-kit/pull/116 https://github.com/liteflow-labs/starter-kit/pull/117

### Description

Adds the collection detail listing page

### How to test

Visit one of the collections and view the listing, the new token UI branch is merged into this branch therefore you should be able to click on a collection from any of the token in the explore page
